### ISSUE TITLE
Need documentation on Hero transitions

### ISSUE THREAD

**Issue by mpcomplete** [flutter/flutter#4517]
There's some good comments in the source, but the [API docs](
http://docs.flutter.io/flutter/widgets/Hero-class.html) are empty.

**Comment by mpcomplete** [comment]
cc @Hixie

**Comment by Hixie** [comment]
Yeah, this is already on my list (on my whiteboard next to my desk). Thanks for filing a real issue. :-)
